blocks/library/shortcode/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { withInstanceId, Dashicon } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlockType, source } from '../../api';
+
+const { text } = source;
+
+registerBlockType( 'core/shortcode', {
+	title: __( 'Shortcode' ),
+
+	icon: 'editor-code',
```

Comment:
@jasmussen could we get a special icon here?

Reply:
@mtias @jasmussen Regarding a special icon. See https://github.com/WordPress/dashicons/issue/261.

Reply:
I'll put this on my list, thanks for the reminder!